boshio/http_client_test.go

```diff
@@ -1,44 +1,20 @@
 package boshio_test
 
 import (
-	"errors"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"strings"
 	"time"
 
 	"github.com/concourse/bosh-io-stemcell-resource/boshio"
+	"github.com/concourse/bosh-io-stemcell-resource/fakes"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
-type tempError struct {
```

Comment:
We are still using a fake so the test time shouldn't increase: https://github.com/ljfranklin/bosh-io-stemcell-resource/blob/0f7abe01a7333261d2660159b4fe97cbc36bc581/boshio/http_client_test.go#L74.